engine/lib/phx/src/ui/hmgui/gui.rs

```diff
@@ -436,12 +436,12 @@ impl HmGui {
 
     /// Add multiline styled text element.
     pub fn text_view(&mut self, text_data: &mut TextData, editable: bool) {
-        let image_item = HmGuiImage {
-            image: std::ptr::null_mut(),
-            layout: HmGuiImageLayout::TopLeft,
-        };
+        // let image_item = HmGuiImage {
```

Comment:
Why this is disabled?

Reply:
I've modified `WidgetItem::TextView` to take an `Option<HmGuiImage>` instead, so this wasn't needed. I should've deleted this though.